modules/infra/digitalocean/outputs.tf

```diff
@@ -20,6 +20,17 @@ output "droplet_ips" {
   ]
 }
 
+output "droplet_ids" {
```

Comment:
Same.

Reply:
This output isn't dependent on the new loadbalancer changes. I added this output for droplet ids to support additional digitalocean resources in recipes that can reference droplets created using this module. This can also be accomplished using the `user:` tags, but using droplet ids is a cleaner approach that avoids referencing unwanted droplets when tags are reused.